scene/2d/skeleton_2d.cpp

```diff
@@ -153,16 +154,6 @@ void Bone2D::_notification(int p_what) {
 			}
 #endif // TOOLS_ENABLED
 		} break;
-
-		case NOTIFICATION_MOVED_IN_PARENT: {
-			if (skeleton) {
-				skeleton->_make_bone_setup_dirty();
-			}
-			if (copy_transform_to_cache) {
-				cache_transform = get_transform();
-			}
-		} break;
-
```

Comment:
Nitpick, but we try to keep one line between each `case` block for `_notification`, as those blocks are usually huge function-like scopes so it helps to separate them a bit.

Reply:
alright, will fix